ui/apps/platform/src/Containers/Compliance/Dashboard/ComplianceDashboardPage.tsx

```diff
@@ -48,24 +49,28 @@ function ComplianceDashboardPage(): ReactElement {
     const hasManageStandardsButton =
         hasWriteAccessForComplianceStandards && isDisableComplianceStandardsEnabled;
 
-    useEffect(() => {
+    function clickManageStandardsButton() {
         setIsFetchingStandards(true);
         fetchComplianceStandardsSortedByName()
             .then((standardsFetched) => {
-                setErrorMessageForStandards('');
+                setErrorMessageFetching('');
                 setStandards(standardsFetched);
+                setIsManageStandardsModalOpen(true);
             })
             .catch((error) => {
-                setErrorMessageForStandards(getAxiosErrorMessage(error));
+                setErrorMessageFetching(getAxiosErrorMessage(error));
                 setStandards([]);
             })
             .finally(() => {
                 setIsFetchingStandards(false);
             });
-    }, []);
+    }
+
+    function onCloseManageStandardsError() {
+        setErrorMessageFetching('');
+    }
 
-    function onSaveManageStandardsModal(standardsSaved: ComplianceStandardMetadata[]) {
-        setStandards(standardsSaved);
+    function onSaveOrCloseManageStandardsModal() {
```

Comment:
nit: I seem to remember _Code Complete_ cautioned that names with "and" or "or" in them were a red flag, because it often indicated the function or concept was overloaded or trying to do too much.

I don't think that's true here, but it's still a little confusing a name. Consider avoiding `Or` in the future.

Reply:
Aha, I will change the contrast to `onCancel` versus `onChange` for either save or close that made changes.